setup.cfg

```diff
@@ -1,2 +1,6 @@
 [bdist_wheel]
 universal=1
+
+[codespell]
+skip = *.c,.riot,.tox,.mypy_cache,ddtrace/vendor
+ignore-words-list = dne,fo,medias,ment,nin,ot,setttings,statics
```

Comment:
Do we still need all these ignores if we ignore the vendor directory?

`setttings` or `nin` seems like ones that shouldn't be ignored

Reply:
<img width="816" alt="Screen Shot 2021-06-15 at 12 38 50 PM" src="https://user-images.githubusercontent.com/35776586/122091293-a6cbc680-cdd6-11eb-8df3-926b8509bc47.png">

Seems like these are all valid used words in our testing and integrations 😢

Reply:
can we either do a PR to fix those usages first, or I am ok to merge this PR as-is, and then update to remove the skips immediately after.